rhcs5_odf_external/README.md

```diff
@@ -135,7 +135,7 @@ The cephadm utility consists of two main components:
 
 ### Registering the Red Hat Ceph Storage nodes ###
 
-Registering the Red Hat Ceph Storage nodes to the CDN and attaching subscriptions. Please ensure that you have the following repositories enabled and correctly as part of your Red Hat subscription entitlements on all your hosts.
+In order to register the Red Hat Ceph Storage nodes to the CDN and to attach subscriptions we need to nake sure that we have the following repositories enabled as part of our Red Hat subscription entitlements on all the hosts in our lab enviroment.
```

Comment:
In order to register the Red Hat Ceph Storage nodes to the CDN and to attach subscriptions, we need to make sure that we have the following repositories enabled as part of our Red Hat subscription entitlements on all the hosts in our lab environment: